src/core.c/Order.pm6

```diff
@@ -54,6 +54,16 @@ multi sub infix:<cmp>(int $a, int $b) {
     ORDER(nqp::cmp_i($a, $b))
 }
 
+multi sub infix:<cmp>(Code:D \a, Code:D \b) {
+    a.name cmp b.name
+}
+multi sub infix:<cmp>(Code:D \a, \b) {
+    a.name cmp b.Stringy
+}
+multi sub infix:<cmp>(Code:D \a, \b) {
```

Comment:
Isn't the Code:D type constraint on the wrong arg here?

Reply:
Yikes. Correct. I'll amend